### PR TITLE
[1LP][RFR] fauxfactory args

### LINKED ISSUE #9135: Update use of fauxfactory to include new kwargs
Changes in fauxfactory use-case as per PR

https://github.com/omaciel/fauxfactory/pull/103

### PR DESCRIPTION
**Pull request by digitronik**
## Purpose or Intent
- Removed string formatting with fauxfactory and move to `start` arg
- Improved some know attributes

Main intention of this PR to move `.format` usecase with fauxfactory

fixes #9135

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit fd6da639e63cab4f20748decbbec45da79cd0bb6

<details>
<summary>
<b>Show fixtures</b>
</summary>


The *local* fixture **`ansible_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**
    - *policy_for_testing*
    - *test_action_run_ansible_playbook_localhost*
    - *test_action_run_ansible_playbook_manual_address*
    - *test_action_run_ansible_playbook_target_machine*

The *local* fixture **`policy_for_testing`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**

The *local* fixture **`ansible_credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`management_event_method`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *management_event_instance*

The *local* fixture **`management_event_instance`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert*

The *local* fixture **`custom_vm_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *test_embedded_ansible_custom_button_localhost*
    - *test_embedded_ansible_custom_button_target_machine*
    - *test_embedded_ansible_custom_button_specific_hosts*

The *local* fixture **`alert`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert_profile*

The *local* fixture **`repository`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_rest.py**
    - *test_edit_repository*
    - *test_delete_repository_from_detail*
    - *test_delete_repository_from_collection*
    - *test_payloads_collection*
    - *test_authentications_subcollection*
    - *test_payloads_subcollection*

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`ansible_policy_linked_vm`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`custom_service_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_tag_ansible_credential*
    - *test_tagvis_ansible_credential*

</details>

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 3a6d2f90c860b87fb2459e4fec6f66256fc83d02

<details>
<summary>
<b>Show fixtures</b>
</summary>


The *local* fixture **`ansible_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**
    - *policy_for_testing*
    - *test_action_run_ansible_playbook_localhost*
    - *test_action_run_ansible_playbook_manual_address*
    - *test_action_run_ansible_playbook_target_machine*

The *local* fixture **`policy_for_testing`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**

The *local* fixture **`ansible_credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`management_event_method`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *management_event_instance*

The *local* fixture **`management_event_instance`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert*

The *local* fixture **`custom_vm_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *test_embedded_ansible_custom_button_localhost*
    - *test_embedded_ansible_custom_button_target_machine*
    - *test_embedded_ansible_custom_button_specific_hosts*

The *local* fixture **`alert`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert_profile*

The *local* fixture **`repository`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_rest.py**
    - *test_edit_repository*
    - *test_delete_repository_from_detail*
    - *test_delete_repository_from_collection*
    - *test_payloads_collection*
    - *test_authentications_subcollection*
    - *test_payloads_subcollection*

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`ansible_policy_linked_vm`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`custom_service_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_tag_ansible_credential*
    - *test_tagvis_ansible_credential*

The *local* fixture **`button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_vm_custom_button.py**
    - *test_custom_button_display_service_vm*
    - *test_custom_button_automate_service_vm*

The *local* fixture **`setup_groups_buttons`** is used in the following files:
- **cfme/tests/automate/custom_button/test_import_export.py**
    - *test_custom_button_import_export*

The *local* fixture **`serv_button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
    - *test_custom_button_text_display*

The *local* fixture **`vis_enb_button`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**

The *local* fixture **`setup_dynamic_dialog`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_vm_custom_button.py**

</details>

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit da32d72e071ef7cb41ab56aa69a9fc887b9f0499

<details>
<summary>
<b>Show fixtures</b>
</summary>


The *local* fixture **`ansible_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**
    - *policy_for_testing*
    - *test_action_run_ansible_playbook_localhost*
    - *test_action_run_ansible_playbook_manual_address*
    - *test_action_run_ansible_playbook_target_machine*

The *local* fixture **`policy_for_testing`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**

The *local* fixture **`ansible_credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`management_event_method`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *management_event_instance*

The *local* fixture **`management_event_instance`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert*

The *local* fixture **`custom_vm_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *test_embedded_ansible_custom_button_localhost*
    - *test_embedded_ansible_custom_button_target_machine*
    - *test_embedded_ansible_custom_button_specific_hosts*

The *local* fixture **`alert`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert_profile*

The *local* fixture **`repository`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_rest.py**
    - *test_edit_repository*
    - *test_delete_repository_from_detail*
    - *test_delete_repository_from_collection*
    - *test_payloads_collection*
    - *test_authentications_subcollection*
    - *test_payloads_subcollection*

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`ansible_policy_linked_vm`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`custom_service_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_tag_ansible_credential*
    - *test_tagvis_ansible_credential*

The *local* fixture **`button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_vm_custom_button.py**
    - *test_custom_button_display_service_vm*
    - *test_custom_button_automate_service_vm*

The *local* fixture **`setup_groups_buttons`** is used in the following files:
- **cfme/tests/automate/custom_button/test_import_export.py**
    - *test_custom_button_import_export*

The *local* fixture **`serv_button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
    - *test_custom_button_text_display*

The *local* fixture **`vis_enb_button`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**

The *local* fixture **`setup_dynamic_dialog`** is used in the following files:
- **cfme/tests/automate/test_service_automate.py**
    - *test_automate_method_with_dialog*

The *local* fixture **`gen_obj_def_import_export`** is used in the following files:
- **cfme/tests/automate/generic_objects/test_definitions.py**
    - *test_import_export_generic_object_definition*

The *local* fixture **`button_with_dialog`** is used in the following files:
- **cfme/tests/automate/generic_objects/test_instances.py**
    - *test_generic_object_with_service_button*

The *local* fixture **`new_user`** is used in the following files:
- **cfme/tests/automate/test_git_import.py**
    - *test_refresh_git_current_user*

The *local* fixture **`generic_object_definition`** is used in the following files:
- **cfme/tests/automate/test_method.py**

The *local* fixture **`new_users`** is used in the following files:
- **cfme/tests/automate/test_service_automate.py**

</details>

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 58302d1b05f1f430adc8d0c0b2f8902163a0c888

<details>
<summary>
<b>Show fixtures</b>
</summary>


The *local* fixture **`ansible_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**
    - *policy_for_testing*
    - *test_action_run_ansible_playbook_localhost*
    - *test_action_run_ansible_playbook_manual_address*
    - *test_action_run_ansible_playbook_target_machine*

The *local* fixture **`policy_for_testing`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_actions.py**

The *local* fixture **`ansible_credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`management_event_method`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *management_event_instance*

The *local* fixture **`management_event_instance`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert*

The *local* fixture **`custom_vm_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *test_embedded_ansible_custom_button_localhost*
    - *test_embedded_ansible_custom_button_target_machine*
    - *test_embedded_ansible_custom_button_specific_hosts*

The *local* fixture **`alert`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_automate.py**
    - *alert_profile*

The *local* fixture **`repository`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_rest.py**
    - *test_edit_repository*
    - *test_delete_repository_from_detail*
    - *test_delete_repository_from_collection*
    - *test_payloads_collection*
    - *test_authentications_subcollection*
    - *test_payloads_subcollection*

The *local* fixture **`ansible_linked_vm_action`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**
    - *ansible_policy_linked_vm*

The *local* fixture **`ansible_policy_linked_vm`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`custom_service_button`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_services.py**

The *local* fixture **`credential`** is used in the following files:
- **cfme/tests/ansible/test_embedded_ansible_tags.py**
    - *test_tag_ansible_credential*
    - *test_tagvis_ansible_credential*

The *local* fixture **`button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_vm_custom_button.py**
    - *test_custom_button_display_service_vm*
    - *test_custom_button_automate_service_vm*

The *local* fixture **`setup_groups_buttons`** is used in the following files:
- **cfme/tests/automate/custom_button/test_import_export.py**
    - *test_custom_button_import_export*

The *local* fixture **`serv_button_group`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**
    - *test_custom_button_text_display*

The *local* fixture **`vis_enb_button`** is used in the following files:
- **cfme/tests/automate/custom_button/test_service_objects.py**

The *local* fixture **`setup_dynamic_dialog`** is used in the following files:
- **cfme/tests/automate/test_service_automate.py**
    - *test_automate_method_with_dialog*

The *local* fixture **`gen_obj_def_import_export`** is used in the following files:
- **cfme/tests/automate/generic_objects/test_definitions.py**
    - *test_import_export_generic_object_definition*

The *local* fixture **`button_with_dialog`** is used in the following files:
- **cfme/tests/automate/generic_objects/test_instances.py**
    - *test_generic_object_with_service_button*

The *local* fixture **`new_user`** is used in the following files:
- **cfme/tests/services/test_different_dialogs_in_catalogs.py**
    - *test_should_be_able_to_access_services_requests_as_user*

The *local* fixture **`generic_object_definition`** is used in the following files:
- **cfme/tests/automate/test_method.py**

The *local* fixture **`new_users`** is used in the following files:
- **cfme/tests/automate/test_service_automate.py**

The *local* fixture **`new_tenant`** is used in the following files:
- **cfme/tests/infrastructure/test_quota.py**
    - *set_parent_tenant_quota*
    - *new_group_list*

The *local* fixture **`new_child`** is used in the following files:
- **cfme/tests/infrastructure/test_child_tenant.py**
    - *set_child_tenant_quota*
    - *new_group*

The *local* fixture **`new_group_child`** is used in the following files:
- **cfme/tests/cloud/test_quota.py**
    - *new_user_child*

The *local* fixture **`new_user_child`** is used in the following files:
- **cfme/tests/cloud/test_quota.py**

The *local* fixture **`new_project`** is used in the following files:
- **cfme/tests/infrastructure/test_project_quota.py**
    - *set_project_quota*
    - *new_group*

The *local* fixture **`new_group_project`** is used in the following files:
- **cfme/tests/cloud/test_quota.py**
    - *new_user_project*

The *local* fixture **`new_user_project`** is used in the following files:
- **cfme/tests/cloud/test_quota.py**

The *local* fixture **`dialog`** is used in the following files:
- **cfme/tests/services/test_dynamicdd_dialogelement.py**
    - *test_dynamicdropdown_dialog*

The *local* fixture **`domain`** is used in the following files:
- **cfme/tests/infrastructure/test_quota_tagging.py**
    - *max_quota_test_instance*

The *local* fixture **`catalog_item`** is used in the following files:
- **cfme/tests/ssui/test_ssui_service_dialog.py**
    - *test_tag_dialog_catalog_item_ssui*

The *local* fixture **`tenant`** is used in the following files:
- **cfme/tests/cloud/test_tenant.py**
    - *test_tenant_crud*

The *local* fixture **`child_tenant`** is used in the following files:
- **cfme/tests/configure/test_access_control.py**
    - *new_tenant_admin*
    - *test_tenantadmin_group_crud*

The *local* fixture **`vm_crud`** is used in the following files:
- **cfme/tests/intelligence/test_reports_with_timelines.py**
    - *setup_for_reports*
    - *generate_policy_event*

The *local* fixture **`ssa_compliance_policy`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_instance_analysis.py**
    - *ssa_compliance_profile*

The *local* fixture **`ssa_compliance_profile`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_instance_analysis.py**
    - *test_ssa_compliance*

The *local* fixture **`ssa_action`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_instance_analysis.py**
    - *ssa_policy*

The *local* fixture **`ssa_policy`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_instance_analysis.py**
    - *assign_profile_to_vm*

The *local* fixture **`schedule_ssa`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_instance_analysis.py**
    - *test_ssa_schedule*

The *local* fixture **`role_only_user_owned`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_ownership.py**
    - *group_only_user_owned*

The *local* fixture **`group_only_user_owned`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_ownership.py**
    - *user1*
    - *user2*
    - *new_user*

The *local* fixture **`role_user_or_group_owned`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_ownership.py**
    - *group_user_or_group_owned*

The *local* fixture **`group_user_or_group_owned`** is used in the following files:
- **cfme/tests/cloud_infra_common/test_vm_ownership.py**
    - *user3*

The *local* fixture **`new_tenant_admin`** is used in the following files:
- **cfme/tests/configure/test_access_control.py**
    - *test_tenantadmin_group_crud*

The *local* fixture **`catalog_obj`** is used in the following files:
- **cfme/tests/configure/test_access_control.py**
    - *test_tenant_unique_catalog*

The *local* fixture **`schedule`** is used in the following files:
- **cfme/tests/configure/test_paginator.py**
    - *test_paginator_details_page*

The *local* fixture **`compute_rate`** is used in the following files:
- **cfme/tests/containers/test_chargeback.py**
    - *assign_compute_rate*

The *local* fixture **`policy_name`** is used in the following files:
- **cfme/tests/control/test_compliance.py**
    - *policy_for_testing*

The *local* fixture **`policy_profile_name`** is used in the following files:
- **cfme/tests/control/test_compliance.py**
    - *policy_for_testing*
    - *assign_policy_for_testing*

The *local* fixture **`host_policy`** is used in the following files:
- **cfme/tests/control/test_actions.py**
    - *test_action_prevent_host_ssa*

The *local* fixture **`vm_compliance_policy_profile`** is used in the following files:
- **cfme/tests/control/test_bugs.py**

The *local* fixture **`compliance_vm`** is used in the following files:
- **cfme/tests/control/test_compliance.py**
    - *test_check_package_presence*
    - *test_check_files*

The *local* fixture **`new_group`** is used in the following files:
- **cfme/tests/infrastructure/test_project_quota.py**
    - *new_user*

The *local* fixture **`vm_name`** is used in the following files:
- **cfme/tests/infrastructure/test_pxe_provisioning.py**
    - *test_pxe_provision_from_template*

The *local* fixture **`new_role`** is used in the following files:
- **cfme/tests/infrastructure/test_project_quota.py**
    - *new_group*

The *local* fixture **`prov_data`** is used in the following files:
- **cfme/tests/infrastructure/test_provisioning_dialog.py**
    - *test_change_cpu_ram*
    - *test_disk_format_select*
    - *test_power_on_or_off_after_provision*
    - *test_tag*
    - *test_provisioning_schedule*
    - *test_provisioning_vnic_profiles*
    - *test_provision_vm_with_2_nics*
    - *test_vmware_default_placement*
    - *test_linked_clone_default*

The *local* fixture **`new_group_list`** is used in the following files:
- **cfme/tests/infrastructure/test_quota.py**
    - *new_user*

The *local* fixture **`catalog_bundle`** is used in the following files:
- **cfme/tests/services/test_catalog_item.py**
    - *test_tagvis_catalog_bundle*

The *local* fixture **`vm`** is used in the following files:
- **cfme/tests/infrastructure/test_scvmm_specific.py**
    - *test_no_dvd_ruins_refresh*
    - *test_check_disk_allocation_size_scvmm*

The *local* fixture **`ssa_analysis_profile`** is used in the following files:
- **cfme/tests/infrastructure/test_ssa_vddk.py**
    - *vm*

The *local* fixture **`tenants_setup`** is used in the following files:
- **cfme/tests/infrastructure/test_tenant_quota.py**
    - *test_setting_child_quota_more_than_parent*

The *local* fixture **`clone_vm_name`** is used in the following files:
- **cfme/tests/infrastructure/test_vm_clone.py**
    - *test_vm_clone*
    - *test_vm_clone_neg*

The *local* fixture **`create_vm`** is used in the following files:
- **cfme/tests/infrastructure/test_vm_clone.py**
    - *test_vm_clone*
    - *test_vm_clone_neg*

The *local* fixture **`local_group`** is used in the following files:
- **cfme/tests/integration/test_cfme_auth.py**
    - *local_user*
    - *test_login_local_group*

The *local* fixture **`user`** is used in the following files:
- **cfme/tests/integration/test_db_auth.py**
    - *test_db_user_pwd*

The *local* fixture **`nonexistent_user`** is used in the following files:
- **cfme/tests/integration/test_db_auth.py**
    - *test_login_invalid_user*

The *local* fixture **`vm_ownership`** is used in the following files:
- **cfme/tests/intelligence/reports/test_validate_chargeback_report.py**
    - *resource_usage*
    - *chargeback_report_default*
    - *chargeback_report_custom*

The *local* fixture **`new_compute_rate`** is used in the following files:
- **cfme/tests/ssui/test_ssui_dashboard.py**
    - *assign_chargeback_rate*

The *local* fixture **`network`** is used in the following files:
- **cfme/tests/networks/test_provision_to_virtual_network.py**

The *local* fixture **`secgroup_with_rule`** is used in the following files:
- **cfme/tests/networks/test_sdn_inventory_collection.py**
    - *test_sdn_nsg_firewall_rules*

The *local* fixture **`sec_group`** is used in the following files:
- **cfme/tests/networks/test_sdn_security_groups.py**
    - *test_security_group_crud*

The *local* fixture **`private_flavor`** is used in the following files:
- **cfme/tests/openstack/cloud/test_flavors.py**
    - *instance_with_private_flavor*
    - *test_flavors_details_from_list_view*
    - *test_flavor_details*

The *local* fixture **`volume`** is used in the following files:
- **cfme/tests/storage/test_volume_snapshot.py**
    - *snapshot*
    - *test_storage_snapshot_create_cancelled_validation*
    - *test_storage_snapshot_create_reset_validation*
    - *test_storage_volume_snapshot_crud*

The *local* fixture **`volume_with_type`** is used in the following files:
- **cfme/tests/openstack/cloud/test_volumes.py**
    - *test_create_volume_with_type*
    - *test_edit_volume_with_type*
    - *test_delete_volume_with_type*

The *local* fixture **`volume_backup`** is used in the following files:
- **cfme/tests/openstack/cloud/test_volume_backup.py**
    - *incremental_backup*
    - *attached_volume*
    - *test_create_volume_backup*
    - *test_restore_volume_backup*

The *local* fixture **`volume_backup_with_type`** is used in the following files:
- **cfme/tests/openstack/cloud/test_volume_backup.py**
    - *test_create_backup_of_volume_with_type*

The *local* fixture **`incremental_backup`** is used in the following files:
- **cfme/tests/openstack/cloud/test_volume_backup.py**
    - *test_create_volume_incremental_backup*
    - *test_restore_incremental_backup*

The *local* fixture **`new_instance`** is used in the following files:
- **cfme/tests/openstack/cloud/test_volumes.py**
    - *test_volume_attach_detach_instance*

The *local* fixture **`temp_extdb_pod_appliance`** is used in the following files:
- **cfme/tests/pod/test_appliance_crud.py**
    - *test_crud_pod_appliance_ext_db*

The *local* fixture **`temp_pod_ansible_appliance`** is used in the following files:
- **cfme/tests/pod/test_appliance_crud.py**

The *local* fixture **`new_vm`** is used in the following files:
- **cfme/tests/services/test_add_remove_vm_to_service.py**
    - *test_add_vm_to_service*

The *local* fixture **`copy_domain`** is used in the following files:
- **cfme/tests/services/test_add_remove_vm_to_service.py**
    - *test_add_vm_to_service*

The *local* fixture **`dialog_cat_item`** is used in the following files:
- **cfme/tests/services/test_dialog_regex_validation_in_catalog.py**
    - *test_dialog_element_regex_validation*

The *local* fixture **`generic_catalog_item`** is used in the following files:
- **cfme/tests/services/test_different_dialogs_in_catalogs.py**
    - *test_dialog_elements_should_display_default_value*

The *local* fixture **`catalog`** is used in the following files:
- **cfme/tests/ssui/test_ssui_service_dialog.py**
    - *catalog_item*

The *local* fixture **`dialog_name`** is used in the following files:
- **cfme/tests/services/test_provision_stack.py**
    - *template*
    - *catalog_item*

The *local* fixture **`template`** is used in the following files:
- **cfme/tests/services/test_provision_stack.py**
    - *catalog_item*
    - *test_remove_non_read_only_orch_template*
    - *test_remove_read_only_orch_template_neg*

The *local* fixture **`new_domain`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**
    - *modified_instance*

The *local* fixture **`user_auth`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**
    - *user_api*

The *local* fixture **`automate_env_setup`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**
    - *get_service_template*

The *local* fixture **`get_service_template`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**

The *local* fixture **`create_domain`** is used in the following files:
- **cfme/tests/services/test_service_manual_approval.py**
    - *modify_instance*

The *local* fixture **`tagcontrol_dialog`** is used in the following files:
- **cfme/tests/ssui/test_ssui_service_dialog.py**
    - *catalog_item*

The *local* fixture **`container`** is used in the following files:
- **cfme/tests/storage/test_object_store_container.py**
    - *test_storage_object_store_container_add_remove_tag*

The *local* fixture **`storage_object`** is used in the following files:
- **cfme/tests/storage/test_object_store_object.py**
    - *test_storage_object_store_object_add_remove_tag*

The *local* fixture **`instance_fixture`** is used in the following files:
- **cfme/tests/storage/test_volume.py**
    - *attached_volume*
    - *test_storage_volume_attach_detach*
    - *test_storage_volume_attached_delete*

The *local* fixture **`backup`** is used in the following files:
- **cfme/tests/storage/test_volume_backup.py**
    - *test_storage_volume_backup_create*
    - *test_storage_volume_backup_edit_tag_from_detail*
    - *test_storage_volume_backup_delete*
    - *test_storage_volume_backup_restore*
    - *test_storage_volume_backup_restore_from_backup_page*

The *local* fixture **`snapshot`** is used in the following files:
- **cfme/tests/storage/test_volume_snapshot.py**
    - *test_storage_volume_snapshot_edit_tag_from_detail*

The *local* fixture **`cancel_migration_plan`** is used in the following files:
- **cfme/tests/v2v/test_v2v_cancel_migrations.py**
    - *test_retry_migration_plan*

</details>

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/cloud_infra_common/test_vm_ownership.py [hunk 1]
```diff
@@ -120,7 +123,7 @@ def user3(appliance, group_user_or_group_owned):
 
 def new_user(appliance, group_only_user_owned):
     user = appliance.collections.users.create(
-        name='user_' + fauxfactory.gen_alphanumeric(),
+        name=fauxfactory.gen_alphanumeric(start="user_"),
         credential=new_credential(),
         email='abc@redhat.com',
         groups=[group_only_user_owned],
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_ownership.py]: Can we use ```fauxfactory.gen_email()``` here for ```email```?

File: cfme/tests/infrastructure/test_provisioning_dialog.py [hunk 2]
```diff
@@ -50,12 +50,10 @@ def vm_name():
 def prov_data(provisioning, provider):
     data = {
         'request': {
-            'email': "{}@{}.test".format(fauxfactory.gen_alphanumeric(),
-                                         fauxfactory.gen_alphanumeric()),
+            'email': fauxfactory.gen_email(),
             'first_name': fauxfactory.gen_alphanumeric(),
```
Comment (ganeshhubale) [inline cfme/tests/infrastructure/test_provisioning_dialog.py]: :100:

File: cfme/tests/intelligence/chargeback/test_resource_allocation.py [hunk 3]
```diff
@@ -75,7 +75,7 @@ def vm_ownership(enable_candu, provider, appliance):
     cb_group = group_collection.instantiate(description='EvmGroup-user')
     user = appliance.collections.users.create(
         name="{}_{}".format(provider.name, fauxfactory.gen_alphanumeric()),
-        credential=Credential(principal='uid{}'.format(fauxfactory.gen_alphanumeric()),
```
Comment (ganeshhubale) [inline cfme/tests/intelligence/chargeback/test_resource_allocation.py]: I think, you missed variable: **name**!

File: cfme/utils/appliance/__init__.py [hunk 4]
```diff
@@ -522,7 +522,7 @@ def _screenshot_capture_at_context_leave(self, exc_type, exc_val, exc_tb):
             short_tb = "{}: {}".format(exc_type.__name__, str(exc_val))
             full_tb = "{}\n{}".format(full_tb, short_tb)
 
-            g_id = "appliance-cm-screenshot-{}".format(fauxfactory.gen_alpha(length=6))
+            g_id = fauxfactory.gen_alpha(length=6, start="appliance-cm-screenshot-")
```
Comment (ganeshhubale) [inline cfme/utils/appliance/__init__.py]: Need to increase ```length``` for **g_id**
```
ipdb> fauxfactory.gen_alpha(length=6, start="appliance-cm-screenshot-")                                                                                                      
'applia'
```

**Comment by digitronik** [comment]
@ganeshhubale @valaparthvi thanks for review.

**Comment by valaparthvi** [comment]
@digitronik I guess you can move this to 1LP now.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**